runners.go

```diff
@@ -384,14 +393,17 @@ func (s *RunnersService) ListGroupsRunners(gid interface{}, opt *ListGroupsRunne
 // GitLab API docs:
 // https://docs.gitlab.com/ce/api/runners.html#register-a-new-runner
 type RegisterNewRunnerOptions struct {
-	Token          *string                       `url:"token" json:"token"`
-	Description    *string                       `url:"description,omitempty" json:"description,omitempty"`
-	Info           *RegisterNewRunnerInfoOptions `url:"info,omitempty" json:"info,omitempty"`
-	Active         *bool                         `url:"active,omitempty" json:"active,omitempty"`
-	Locked         *bool                         `url:"locked,omitempty" json:"locked,omitempty"`
-	RunUntagged    *bool                         `url:"run_untagged,omitempty" json:"run_untagged,omitempty"`
-	TagList        *[]string                     `url:"tag_list[],omitempty" json:"tag_list,omitempty"`
-	MaximumTimeout *int                          `url:"maximum_timeout,omitempty" json:"maximum_timeout,omitempty"`
+	Token           *string                       `url:"token" json:"token"`
+	Description     *string                       `url:"description,omitempty" json:"description,omitempty"`
+	Info            *RegisterNewRunnerInfoOptions `url:"info,omitempty" json:"info,omitempty"`
+	Active          *bool                         `url:"active,omitempty" json:"active,omitempty"`
+	Paused          *bool                         `url:"paused,omitempty" json:"paused,omitempty"`
+	Locked          *bool                         `url:"locked,omitempty" json:"locked,omitempty"`
+	RunUntagged     *bool                         `url:"run_untagged,omitempty" json:"run_untagged,omitempty"`
+	TagList         *[]string                     `url:"tag_list[],omitempty" json:"tag_list,omitempty"`
+	AccessLevel     *string                       `url:"access_level,omitempty" json:"access_level,omitempty"`
+	MaximumTimeout  *int                          `url:"maximum_timeout,omitempty" json:"maximum_timeout,omitempty"`
+	MaintenanceNote *string                       `url:"maintenance_note,omitempty" json:"maintenance_note,omitempty"`
 }
```

Comment:
Note: The reason these all flag as "changed" even though only 3 rows were added (Paused, AccessLevel, MaintenanceNote) is because I had to add one space to all the other rows to maintain formatting since the MaintenanceNote variable name was 1 character longer than Maximum Timeout 😄 

Easy to miss, so I figured I'd leave a comment.